src/main/java/teammates/common/util/SanitizationHelper.java

```diff
@@ -372,4 +372,23 @@ public static boolean isSanitizedHtml(String string) {
     public static String desanitizeIfHtmlSanitized(String string) {
         return isSanitizedHtml(string) ? desanitizeFromHtml(string) : string;
     }
+
+    /**
+     * Sanitizes the log message, only allow br element and span element with class attribute equals to
+     * "bold" or "text-danger". Convert other special characters into HTML-safe equivalents.
+     */
+    public static String sanitizeForLogMessage(String unsanitizedString) {
+        if (unsanitizedString == null) {
+            return null;
+        }
+        return unsanitizedString
+                .replaceAll("<(?!(/?(span( class=\"(bold|text-danger)\")?|br)>))", "&lt;")
```

Comment:
I believe there is a more readable way to write this. Check `richTextPolicy` in SanitizationHelper

Reply:
`richTextPolicy` uses the OWASP sanitization library for Java which will remove disallowed elements instead of escaping them. Hence I had to write my own regular expression to do this. I went for conciseness here. The more readable way would be to list out all the full-string possibilities, i.e. something like

```
"<(?!(span class=\"bold\">|span class=\"text-danger\">|/span>|<br>))"
```

Do you think we should use this form instead?